scripts/validate_datetime.py

```diff
@@ -0,0 +1,61 @@
+"""
+This is pure Python implementation of DateTime
```

Comment:
Could you explain in more depth please and also link some relevant documentation